website/docs/migrating-from-docker/using-the-docker_host-environment-variable.md

```diff
@@ -9,14 +9,22 @@ tags: [migrating-from-docker]
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Using the `DOCKER_HOST` environment variable
+# Using the `DOCKER_HOST` environment variable 
 
 Consider using the `DOCKER_HOST` environment variable to migrate transparently from Docker to Podman Desktop on all platforms.
 
 - Continue using familiar Docker commands.
 - Take advantage of the benefits of Podman.
 - Your tools, such as [Gradle](https://gradle.org/) or [Testcontainers](https://www.testcontainers.com/), communicate with Podman without reconfiguration.
 
+Using the `DOCKER_HOST` environment variable, you can connect your Docker CLI to the Podman socket.
+
+:::note
+
+Alternatively, you can add a `podman` context using the `docker context create` command. Then, you can run the `docker context use podman` command to switch to that context. This way you can use your Docker CLI to run your tasks on a remote Podman engine. When you run the `docker ps` command, it queries the Podman socket specified in the current context.
```

Comment:
suggestion: maybe tell that the value of the context should follow the pattern:

`docker context create podman --docker "host=unix://$HOME.local/share/containers/podman/machine/podman.sock"`

(for example on macOS)
so `host=unix://<$DOCKER_HOST value>`